test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.client.cs

```diff
@@ -104,6 +117,7 @@ public void GrpcClientCallsAreCollectedSuccessfully(string baseAddress, bool sho
             Assert.Equal(0, activity.GetTagValue(SemanticConventions.AttributeRpcGrpcStatusCode));
         }
 
+#if !NETFRAMEWORK
```

Comment:
Many of the tests in this suite require an actual end-to-end call using a non-mocked HttpClient. To test the same functionality for .NET Framework, we would need to devise a more complicated integration test. Most likely involving spinning up Windows containers.

I'm proposing we acknowledge the short-comings of this test suite for .NET Framework and we ship a `netstandard2.0` version of the gRPC client instrumentation anyways.

Reply:
Agreed!